deps/rabbitmq_auth_backend_oauth2/src/uaa_jwt.erl

```diff
@@ -51,6 +51,23 @@ update_uaa_jwt_signing_keys(UaaEnv0, SigningKeys) ->
     UaaEnv2 = [{signing_keys, SigningKeys} | UaaEnv1],
     application:set_env(?APP, key_config, UaaEnv2).
 
+-spec update_jwks_signing_keys() -> ok | {error, term()}.
+update_jwks_signing_keys() ->
+    UaaEnv = application:get_env(?APP, key_config, []),
+    case proplists:get_value(jwks_url, UaaEnv) of
+        undefined ->
+            {error, no_jwks_url};
+        JwksUrl ->
+            case httpc:request(JwksUrl) of
+                {ok, {_, _, JwksBody}} ->
+                    KeyList = maps:get(<<"keys">>, jose:decode(erlang:iolist_to_binary(JwksBody)), []),
```

Comment:
Is this response format standard? Where is it documented?

Reply:
Yes, it's part of the JWK RFC: https://tools.ietf.org/html/rfc7517#section-5

Reply:
This is also part of the OpenID Connect spec, including the refresh mechanism: https://openid.net/specs/openid-connect-core-1_0.html#RotateSigKeys